pypeit/par/pypeitpar.py

```diff
@@ -2773,6 +2774,19 @@ def __init__(self, filt_iter=None, sobel_mode=None, edge_thresh=None, follow_spa
                                 '`center_mode` for these edges and place them at the edge of ' \
                                 'the detector (with the relevant shape).'
 
+        defaults['sync_bound'] = True
```

Comment:
I think the default should be False here as per our discussion. 

Also, I think sync_bound is not the best name for this parameter. If I understand correctly, the *only* thing this governs is whether or not a fake left and right slit edge are added at the detector boundaries when no slits are found, is that correct? I would call it maybe "slits_full_detector" or something like that, and I should add that if this parameter is set edgetrace probably should not even try to find slits and should just return the edges consistent with the detector.

Reply:
Changed the default to False, and I changed the name of the parameter to `bound_detector`.  I also added some text to the slit-tracing troubleshooting page (doc/slit_tracing.rst).  I'm running tests now and will restart the dev-suite.  I did get a failure in a previous version that I need to look into.  Please look through and provide any edits on the doc text that I added.  Here's what it looks like when it's formatted:
![tmpdocs](https://user-images.githubusercontent.com/15785898/108787580-e79f8700-752a-11eb-92d0-d6a2c21be571.png)

Reply:
Hi Kyle, 

This looks amazing. I would for 2) add an explanation of when this behaviour is required, i.e. "for instruments for which the entire detector is illuiminated so that the slit edges project to be off the detector, or for situations where the detector is windowed (Kast) in such a way that the slit edges are not covered by the windowed detector."

For the warning, make it more clear that the "Users should be sure to add a standard star to the PypeIt file (and have it classified as standard), in which case PypeIt will use the standard star trace as a crutch for object tracing." 

Joe